doc/Analytics.xml

```diff
@@ -4515,9 +4516,14 @@ xmlns:acme="http://www.acme.com/schema"&gt;
             <para role="param">ConfidenceLevel - optional [xs:float]</para>
             <para role="text">Minimum confidence level of object classification for generating
               aggregation event. </para>
+            <para role="param">EarlyReport - optional [xs:boolean]</para>
+            <para role="text">Signals that a device shall send an intermediate report as soon as it has sufficient information.</para>
```

Comment:
it is initial or early report, the word intermediate report will confuse in this sentence context.